tests/MongoDB.Analyzer.Tests.Common.TestCases/EF/EFBasic.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright 2021-present MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using MongoDB.Analyzer.Tests.Common.DataModel;
+using MongoDB.Driver;
+
+namespace MongoDB.Analyzer.Tests.Common.TestCases.EF;
+
+public sealed class EFBasic
+{
+    public MyDbContext GetDbContext() => new MyDbContext(new DbContextOptionsBuilder<MyDbContext>().Options);
+    public DbContextOptions GetDbContextOptions() => new DbContextOptionsBuilder<MyDbContext>().Options;
+    public DbSet<Customer> GetDbSet_Customers() => new MyDbContext(new DbContextOptionsBuilder<MyDbContext>().Options).Customers;
+    public DbSet<User> GetDbSet_Users() => new MyDbContext(new DbContextOptionsBuilder<MyDbContext>().Options).Users;
+
+
```

Comment:
Remove empty line, and place those helper method at the bottom of the file.

Reply:
Done